src/control/cmd/dmg/pool.go

```diff
@@ -193,8 +195,42 @@ func (d *PoolGetACLCmd) Execute(args []string) error {
 		return err
 	}
 
-	d.log.Infof("Pool-get-ACL command succeeded, UUID: %s\n", d.UUID)
-	d.log.Info(formatACL(resp.ACL))
+	d.log.Debugf("Pool-get-ACL command succeeded, UUID: %s\n", d.UUID)
+
+	acl := formatACL(resp.ACL)
+
+	if d.File != "" {
+		err = d.writeACLToFile(acl)
+		if err != nil {
+			return err
+		}
+		d.log.Infof("Wrote ACL to output file: %s", d.File)
+	} else {
+		d.log.Info(acl)
+	}
+
+	return nil
+}
+
+func (d *PoolGetACLCmd) writeACLToFile(acl string) error {
+	// Keep the user from clobbering existing files
+	_, err := os.Stat(d.File)
+	if err == nil {
```

Comment:
I'm reasonably certain that whatever causes Stat() to fail will also cause Create() to fail. Even in the weird situation where the current user doesn't have read permission on the file but does have write permission, Stat() will succeed, and err will be nil. I wasn't sure about that, but I just wrote a quick test program to verify it.

If we really wanted belt-and-suspenders, this would do it, but I'm not sure it's necessary:
```go
if _, err := os.Stat(d.File); !os.IsNotExist(err) {
        if err != nil {
                return err
        }
        return errors.New(fmt.Sprintf("file already exists: %s", d.File))
}
```

Reply:
I was initially more thorough but it seemed unnecessary - looking at stat(2) and open(2) manpages, it seems like creating a file returns a superset of the failures possible with stat. I can go with the more thorough check if you prefer though.

Reply:
Doesn't seem necessary to me either.

Reply:
Added a test case for the weird case @mjmac mentioned - it works as expected. Leaving the actual check as-is for now.